.github/MAINTAINING.md

```diff
@@ -0,0 +1,88 @@
+
+Other important notes:
+
+- CloudFront service client updates have previously caused an issue when a new field introduced in the SDK was not included with Terraform and caused all requests to error (https://github.com/terraform-providers/terraform-provider-aws/issues/4091). As a precaution, if you see CloudFront updates, run all the CloudFront resource acceptance testing before merging (`TestAccAWSCloudFront`).
+- Occassionally, there will be changes listed in the authentication pieces of the AWS Go SDK codebase, e.g. changes to `aws/session`. The AWS Go SDK `CHANGELOG` should include a relevant description of these changes under a heading such as `SDK Enhancements` or `SDK Bug Fixes`. If they seem worthy of a callout in the Terraform AWS Provider `CHANGELOG`, then upon merging we should include a similar message prefixed with the `provider` subsystem, e.g. `* provider: ...`. Additionally, if a `CHANGELOG` addition seemed appropriate, this dependency and version should also be updated in the Terraform S3 Backend, which currently lives in Terraform Core. An example of this can be found with https://github.com/terraform-providers/terraform-provider-aws/pull/9305 and https://github.com/hashicorp/terraform/pull/22055.
+
+##### Terraform Updates
+
+Run the full acceptance testing suite against the pull request and verify there are no new or unexpected failures.
+
+### Pull Request Merge Process
+
+- Add this pull request to the upcoming release milestone
+- Add any linked issues that will be closed by the pull request to the same upcoming release milestone
+- Merge the pull request
+- Delete the branch (if the branch is on this repository)
+- Update the repository `CHANGELOG.md`. See also the [Extending Terraform documentation](https://www.terraform.io/docs/extend/best-practices/versioning.html) for more information about the expected CHANGELOG format.
+- Leave a comment on any issues closed by the pull request noting that it has been merged and when to expect the release containing it, e.g.
+
+```markdown
+The fix for this has been merged and will release with version X.Y.Z of the Terraform AWS Provider, likely in the XXX timeframe.
+```
+
+## Release Process
+
+- Ensure a milestone exists for the next release after this release (Generally, the next milestone will be a minor version increase unless previously decided for a major or patch version)
+- Check the upcoming release milestone for open items and either work through them or move them to the next milestone
+- Run the HashiCorp (non-OSS) TeamCity release job with the `DEPLOYMENT_TARGET_VERSION` matching the expected release milestone and `DEPLOYMENT_NEXT_VERSION` matching the next release milestone
+- Wait for the TeamCity release job and TeamCity website deployment jobs to complete
```

Comment:
Is there an email or other notification?

Reply:
Yes, there is a start and finish notification in our internal Slack.